docs/index.rst

```diff
@@ -785,6 +785,14 @@ The SecretStr and SecretBytes will be formatted as either `'**********'` or `''`
 
 (This script is complete, it should run "as is")
 
+Strict Types
+............
+
+You can use the ``StrictStr``, ``StrictInt`` and ``StrictFloat`` types to prevent coercion from compatible types.
+These types will only pass validation when the validated value is of the respective type or is a subtype of that type.
+This behavior is also exposed via the ``strict`` field of the ``ConstrainedStr``, ``ConstrainedFloat`` and
+``ConstrainedInt`` classes and can be combined with a multitude of complex validation rules.
```

Comment:
How about mentioning `StrictBool` as well?

> You can use the ``StrictStr``, ``StrictInt``, ``StrictFloat``, and ``StrictBool`` types to prevent coercion from compatible types.
These types will only pass validation when the validated value is of the respective type or is a subtype of that type.
This behavior is also exposed via the ``strict`` field of the ``ConstrainedStr``, ``ConstrainedFloat`` and
``ConstrainedInt`` classes and can be combined with a multitude of complex validation rules (please note that there is no ``ConstrainedBool`` class).

Reply:
If you rewrite this as an actual code suggestion I can commit it straight from the GitHub UI.

Reply:
> If you rewrite this as an actual code suggestion I can commit it straight from the GitHub UI.

Did so, though I think doing it for multi-line text is a nuisance.